src/test/java/com/google/api/codegen/testdata/library_config.baseline

```diff
@@ -16,10 +16,12 @@ language_settings:
 interfaces:
 - name: google.example.library.v1.LibraryService
   collections:
+  - name_pattern: archives/{archif_path=**}/books/{book}
```

Comment:
Hmmm, that link doesn't singularize...
Maybe this one: http://www.java2s.com/Tutorial/Java/0040__Data-Type/Transformswordstosingularpluralhumanizedhumanreadableunderscorecamelcaseorordinalform.htm